apps/container/src/components/SingleComment.vue

```diff
@@ -0,0 +1,231 @@
+<template>
+  <v-container>
+    <v-row align="start" justify="start">
+      <v-col
+        sm=""
+        order="1"
+        class="mr-3 justify-center pa-0 flex-grow-0 flex-shrink-1"
+      >
+        <div
+          :style="conceptStyle"
+          class="text-white d-flex align-center justify-center rounded-lg"
+        >
+          {{ comment?.enrollment.conceito }}
+        </div>
+        <p class="text-primary" style="white-space: nowrap">
+          {{ `Q${season.split(':')[1]} ${season.split(':')[0]}` }}
+        </p>
+      </v-col>
+      <v-col
+        cols="12"
+        sm=""
+        order="3"
+        order-sm="2"
+        class="comment-text-container mr-2 pa-0"
+        :style="`max-height: ${expanded || smAndDown ? 'unset' : '80px'}`"
+      >
+        <div>
+          <div class="d-flex align-center text-primary">
+            <v-chip
+              v-if="isEAD"
+              density="compact"
+              class="px-2 mr-1"
+              style="font-size: 12px"
+              >EAD</v-chip
+            >
+            <p class="text-subtitle-2">
+              {{ comment?.subject.name }}
+            </p>
+          </div>
+
+          <p ref="commentText">
+            {{ comment?.comment }}
+          </p>
+          <button
+            class="show-more text-primary text-subtitle-2"
+            :style="`display:${expanded || smAndDown ? 'none' : 'block'}`"
+            @click="showMore"
+          >
+            MOSTRAR MAIS
+            <v-icon>mdi-chevron-down</v-icon>
+          </button>
+        </div>
+      </v-col>
+      <v-col
+        sm=""
+        order="2"
+        order-sm="3"
+        class="pa-0 justify-start text-subtitle-2 text-next-light-gray flex-grow-0 flex-shrink-1"
+      >
+        <div class="d-flex">
+          <v-btn
+            :loading="isPendingLike || isPendingRemoveLike"
+            @click="like ? mutateRemoveLike() : mutateLike()"
+            class="icon-button text-subtitle-2"
+            variant="text"
+            :icon="like ? 'mdi-thumb-up' : 'mdi-thumb-up-outline'"
+            :color="like ? 'primary' : ''"
+          >
+            <v-icon size="small" class="mr-1" />
+            {{ `${likeCount || 0}` }}
```

Comment:
no proprio ref ja está sendo feito o fallback para 0
```suggestion
            {{ likeCount }}
```